dspace-server-webapp/src/test/java/org/dspace/app/rest/GroupRestRepositoryIT.java

```diff
@@ -1914,5 +1938,987 @@ public void patchNameTest() throws Exception {
+        configurationService.setProperty("core.authorization.collection-admin.admin-group", false);
+        context.restoreAuthSystemState();
+
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(
+            post("/api/eperson/groups/" + adminGroup.getID() + "/epersons")
+                .contentType(parseMediaType
+                    (org.springframework.data.rest.webmvc.RestMediaTypes.TEXT_URI_LIST_VALUE))
+                .content("https://localhost:8080/server/api/eperson/epersons/" + ePerson.getID()))
+                        .andExpect(status().isForbidden());
+
+        token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(get("/api/eperson/groups/" + adminGroup.getID() + "/epersons"))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded.epersons", Matchers.not(Matchers.hasItem(
+                            EPersonMatcher.matchEPersonOnEmail(ePerson.getEmail())
+                        ))));
+
+        context.turnOffAuthorisationSystem();
+        configurationService.setProperty("core.authorization.community-admin.collection.admin-group", true);
+        configurationService.setProperty("core.authorization.collection-admin.admin-group", true);
+        context.restoreAuthSystemState();
+
+    }
+
+    @Test
+    public void collectionAdminRemoveMembersFromCollectionAdminGroupPropertySetToFalse() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+        Group adminGroup = collectionService.createAdministrators(context, collection);
+        authorizeService.addPolicy(context, collection, Constants.ADMIN, eperson);
+        EPerson ePerson = EPersonBuilder.createEPerson(context).withEmail("testToAdd@test.com").build();
+        context.restoreAuthSystemState();
+
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(
+            post("/api/eperson/groups/" + adminGroup.getID() + "/epersons")
+                .contentType(parseMediaType
+                    (org.springframework.data.rest.webmvc.RestMediaTypes
+                         .TEXT_URI_LIST_VALUE))
+                .content("https://localhost:8080/server/api/eperson/epersons/" + ePerson.getID()));
+
+        getClient(token).perform(get("/api/eperson/groups/" + adminGroup.getID() + "/epersons"))
+                        .andExpect(status().isOk())
+                        .andExpect(jsonPath("$._embedded.epersons", Matchers.hasItem(
+                            EPersonMatcher.matchEPersonOnEmail(ePerson.getEmail())
+                        )));
+        context.turnOffAuthorisationSystem();
+        configurationService.setProperty("core.authorization.community-admin.collection.admin-group", false);
+        configurationService.setProperty("core.authorization.collection-admin.admin-group", false);
+        context.restoreAuthSystemState();
```

Comment:
No need to turn off/on authorization to set configs. The context lines here can be removed